charts/taraxa-info-api/templates/configmap.yaml

```diff
@@ -13,4 +13,7 @@ data:
   DELEGATION_API_ROOT_URI: {{ .Values.app.mainnetDelegationUri | quote }}
   COINGECKO_API_TARAXA_URI: {{ .Values.app.coingeckoUri | quote }}
   TESTNET_EXPLORER_API_ROOT_URI: {{ .Values.app.testnetExplorerUri | quote }}
-  EXPLORER_API_ROOT_URI: {{ .Values.app.mainnetExplorerUri | quote }}
```

Comment:
Why did we remove this?

Reply:
My mistake, possibly when merged main back